packages/remix-dev/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@react-router/dev",
-  "version": "2.9.0-pre.0",
+  "version": "6.22.3",
```

Comment:
Needed these to be in the 6.x range so they'll bump to 7.x when changesets does a major bump